Updated server installation and alpha signup page (#48)
nanos-world/docs: docs/core-concepts/server-manual/server-linux-arm.md

---
title: Linux & ARM Server installation
description: Special cases for some Linux os & ARM platform
sidebar_position: 5
tags: [hosting]
---

Special cases for some linux os & ARM platform server installation.

:::caution

The installation on versions on and below Debian 11, Ubuntu 20 and ARM systems is not officially supported and currently maintained by the community! The following steps are WIP and may change at any time!

:::

## Linux Debian 11 Installation

:::warning

> Please be carefull of the scripts and commands you will run!

Reviewer: ```suggestion
Please be careful with scripts and commands, do it at your own risk!
```

:::

If you are running debian 10 or older, please upgrade to debian 11 or change your operating system to at least Ubuntu 22.04.1 LTS

:::tip

This will fix this type of error: `version GLIBCXX 3.4.30 not found` or/and `version GLIBC 2.34 not found`

:::

Since nanos world use `gcc 11`, we will need to upgrade one of our package to the testing repository.

### With a Script

> You can run this script to install the required libraries, but please look at the script code to see what the script will do. (make sure you run it as root!)

Reviewer: ```suggestion
You can run this script to install the required libraries, but please look at the script code to see what the script will do. (make sure you run it as root!)

```
